### PR TITLE
[10.0] [ADD] analytic_resource_plan

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Analytic Resource Planning
==========================

An effective planning of the resources required for a project or analytic account becomes essential in organizations that are run by projects, or profit center accounting. The process of resource planning generally follows an rolling wave planning approach, in which the level of detail of the planned resources increases over time, as the details of the work required are known to the planning group.

Resources planned for a project/analytic account have an impact on the planned costs. If the resources are procured internally, the standard cost is determined. If the resources are procured externally, the user can indicate the supplier, and the planned costs are then determined on the basis of the supplier's price list.

Multiple planning versions can be maintained for a resource plan so that the organization can create a first rough-cut resource plan, that can then be refined as the project progresses.

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
image

Invalid field type in account_analytic_account

File: analytic_resource_plan/model/analytic_resource_plan.py [hunk 1]
```diff
@@ -28,7 +31,7 @@ def _has_child(self):
         'Analytic Account',
         required=True,
         ondelete='cascade',
-        select=True,
+        index=True,
         domain=[('type', '<>', 'view')],
```
Comment (AaronHForgeFlow) [inline analytic_resource_plan/model/analytic_resource_plan.py]: The domain is still here @nikul-serpentcs

**Comment by nikul-serpentcs** [comment]
@aheficent Removed domain, Could you please look it.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Can you add the form view for the resource plan lines? Is missing

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
There's a problem in the on_change method. Can make this module depend on analytic_schedule?

image

**Comment by nikul-serpentcs** [comment]
@aheficent added 'analytic_schedule' in depends, Could you please look it.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**